scripts/docker/requirements-20.04.txt

```diff
@@ -19,3 +19,7 @@ python3-dev=3.8.2-0ubuntu2
 python3.8-venv
 cargo
 ninja-build
+libyaml-cpp-dev=0.6.2-4ubuntu1
```

Comment:
Hmmm do we need all this? Are you sure?

I think we can delete boost and yaml-cpp for example.